Documentation/JSON_Schemas/CreateGame_Response.json

```diff
@@ -0,0 +1,42 @@
+{
+  "type": "object",
+  "properties": {
+    "user": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "integer"
+        }
+      },
+      "required": [
+        "id"
+      ]
+    },
+    "game": {
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "integer"
+        }
+      },
+      "required": [
+        "id"
+      ]
+    }
+  },
+  "required": [
+    "user",
+    "game"
+  ]
+}
+
+Example:
+
+{
+  "user": {
```

Comment:
Returning user ID. I'm not sure how fatal this would have to be before this becomes an issue (the user always has his ID and should have verified this through the request), but it could stay. Doesn't have negative impact